iroh-blobs/src/store/fs.rs

```diff
@@ -2253,7 +2371,11 @@ impl ActorState {
                 tx.send(res).ok();
             }
             ActorMessage::Delete { hashes, tx } => {
```

Comment:
I would have just added the force flag to the Delete message, but this works as well...